ministryofjustice/correspondence_tool_staff
# CT-2863 Trigger "edit case" event on case update

## Description
Added the call to the state machine transition for "edit case" which adds a new Case History entry. Created a ticket for refactoring all this to use the same mechanism as the other Correspondence Types when opportunity permits. 

## Self-review checklist


### Screenshots


![image](https://user-images.githubusercontent.com/1161161/85791654-1c540800-b72a-11ea-91dc-a828540d49b6.png)
### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-2863

### Deployment
n/a

### Manual testing instructions
Log in as Brian Rix - view an Offender SAR case and use the "Change" link to edit some of the details. You should see a new log entry in the Case History table to show that an edit has taken place.
